### ISSUE TITLE
SSL Error

### ISSUE THREAD

**Issue by caitlindaitch** [ga-wdi-exercises/project3#140]
I'm getting this error, which Matt and I spent about 30 minutes trying to figure out how to debug. The rest of my group is able to run our code which I just pulled from GitHub, but I'm getting this error:

`SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed`

We found a hackey way to get around it before, but now that I'm using a group member's code the issue has returned and I feel like this could be a real issue if it keeps coming up. Any clue how to resolve it?

**Comment by mattscilipoti** [comment]
We tried all the salient suggestions here: http://railsapps.github.io/openssl-certificate-verify-failed.html.  

```
$ rvm osx-ssl-certs status all
# Certificates for...
$ rvm osx-ssl-certs update all
# Updating certificates…
```

The call is not verifying the certificate for her.  
As a workaround, you can tell httparty to skip verifying the certificate, see http://stackoverflow.com/a/22695651/232247.  Update that httparty call, commit, and share the change.  
I want to be clear, we are reducing security for this call. However, I feel this is an ok move for a class project.

**Comment by mattscilipoti** [comment]
Idea.  An older version of “ram” may not get the latest copy of the certificates.  Try updating rvm first:

```
$ rvm get latest
$ rvm osx-ssl-certs update all
```

**Comment by caitlindaitch** [comment]
So we have absolutely no idea why this worked, but it did:

rvm reinstall 2.2.0 --disable-binary
gem install bundler
rvm osx-ssl-certs status all
rvm osx-ssl-certs update all

Thans to @mattscilipoti and @RobertAKARobin for suffering through this with me!! You are heroes.

**Comment by mattscilipoti** [comment]
Follow up thoughts: you should be able to reinstall your current version of ruby, instead of an earlier version.

Why?  I think the rebuild ensures you are incorporating/referencing the latest OpenSSL version into your ruby.  This is based on gathering information from multiple conversations about this, not from me diving in and figuring it out - in code.

**Comment by mattscilipoti** [comment]
Why did it appear again? The fix you applied yesterday, was to install another version of ruby (2.0.0), that utilized the right instance of OpenSSL (the one from homebrew, rather than Apple's). As I mentioned yesterday, rvm still defaulted to 2.3.0, and it would switch you back to that - thereby using an ruby that was not "fixed".  I recommend you:
- "fix" 2.3.0, using these instructions (http://stackoverflow.com/a/33035529/232247)
- uninstall ruby 2.0.0 (see rvm.io for instructions).

**Comment by ssatyal** [comment]
It worked for Caitlin, didn't work for me and we ran the same commands.

Still getting the error:
OpenSSL::SSL::SSLError in TripsController#home
SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
